common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -33,6 +33,6 @@
     <div class="script_placeholder" data-src="${previewer}"/>
 
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-describedby="answer_input_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
I wasn't able to figure out how to create a formula input problem that will produce a msg. But I don't think that the value for aria-describedby is correct here, as "answer_input_${id}" does not appear anyplace else in the template.